exasol/python_extension_common/deployment/language_container_validator.py

```diff
@@ -0,0 +1,137 @@
+from __future__ import annotations
+from typing import Generator
+from datetime import timedelta
+import random
+import string
+from contextlib import contextmanager
+from textwrap import dedent
+
+from tenacity import retry
+from tenacity.wait import wait_fixed
+from tenacity.stop import stop_after_delay, stop_after_attempt
+
+import pyexasol     # type: ignore
+
+_DUMMY_UDF_NAME = 'DUMMY_UDF'
+
+
+def _dress_schema(schema: str | None) -> str:
+
+    if schema:
+        return f'"{schema}".'
+    return ''
```

Comment:
I think that if you add this here, the call site becomes very hard to read. Most likely, others will wonder (as I did) if the SQL statement misses a `.`
I suggest that you either provide a function that escapes the full name and adds the schema if required or that you leave the `.` part to the call site. 

e.g.:
```python
def _escape(schema: str | None, object_name: str):
   ....
```

Reply:
I take the point about the apparently missing dot. Leaving the dot in the SQL is not an option, as the object name may not be qualified if the schema is not specified. Will make the function create the full name.